.travis.yml

```diff
@@ -46,6 +51,6 @@ deploy:
     file_glob: true
     file: "netdata*.tar.*"
     on:
-      condition: $CC = gcc
+      condition: $CC = gcc && $TRAVIS_OS_NAME
```

Comment:
@glensc Yes, this is correct. Sorry for the mistake. @ktsaou I can make a pull request but it seems more easy if you fix this in one of your next "minor fix PR". Is that ok for you?